teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
+
+teacher = Teacher.new('Math')
+puts teacher.can_use_services?
```

Comment:
[Optional] As mentioned above and in order to prevent executing these lines of code when the file is imported, kindly remove these lines of code.